### DISCUSSION TITLE
Error: EBUSY: resource busy or locked, lstat 'C:\DumpStack.log.tmp'

### DISCUSSION THREAD

**Discussion by theirongiant** [webpack/webpack#16581] [General]
Hi,

** Just to clarify this is regarding webpack-dev-server, posting here as it doesn't have it's own discussion area

I recently took the time to upgrade to the latest node version and bring all the packages up to their latest versions for a react project I have. Everything seemed fine but every time I ran the dev version after, at most, 20 minutes it would crash with the error in the title. It's been a couple of weeks and I've finally found that it was due to me incorrectly setting the devServer static value to '/public'. I realise this was a case of RTFM and it should have been set to { directory: path.join(__dirname, 'public') } or left out as it'll default to that anyway but given how abstract the error is I was looking to see if there was a way to catch these kind of errors.

I'm not super familiar with the code base but I'm looking at the watchFiles function in lib/Server.js and I'm wondering is there any reason not to add a check to see if the watchPath exists before executing chokidar.watch(watchPath, watchOptions). As far as I understand chokidar will throw this error when you try to watch a path that doesn't exist and I think it would be better throwing a more explicit error during the start up of webpack-dev-server rather than having it throw a fairly vague one at some random point 20 minutes afterwards.

Happy to take advice from more knowledgeable heads before I start hacking a PR.

**Comment by alexander-akait** [comment]
> As far as I understand chokidar will throw this error when you try to watch a path that doesn't exist and I think it would be better throwing a more explicit error during the start up of webpack-dev-server rather than having it throw a fairly vague one at some random point 20 minutes afterwards.

Hello, checking paths increase initial startup time (especial for big projects), that is why it don't do it

**Reply by theirongiant** [reply]
I guess that's fair enough, hopefully it'll be enough for this discussion to appear in google searches for the error message to help anyone  who makes the same dumb mistake I did.
